tests/chopper_test.py

```diff
@@ -0,0 +1,87 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
+
+import scipp as sc
+
+import tof
+
+Hz = sc.Unit('Hz')
+deg = sc.Unit('deg')
+rad = sc.Unit('rad')
+meter = sc.Unit('m')
+
+
+def test_angular_speed():
+    f = 10.0 * Hz
+    chopper = tof.Chopper(
+        frequency=f,
+        open=0.0 * deg,
+        close=10.0 * deg,
+        phase=0.0 * deg,
+        distance=1.0 * meter,
+    )
+    assert chopper.omega == (2.0 * rad * sc.constants.pi) * f
+
+
+def test_open_close_times():
+    f = 10.0 * Hz
+    chopper = tof.Chopper(
+        frequency=f,
+        open=sc.array(dims=['cutout'], values=[10.0], unit='deg'),
+        close=sc.array(dims=['cutout'], values=[20.0], unit='deg'),
+        phase=0.0 * deg,
+        distance=1.0 * meter,
```

Comment:
Having a factor(?) of 1 is bad for tests, since it it hard to tell whether it was missed or accidentally included in an equation.